src/wallet/rpcdump.cpp

```diff
@@ -1352,6 +1352,7 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
             "      \"internal\": <true>                                    , (boolean, optional, default: false) Stating whether matching outputs should be treated as not incoming payments\n"
             "      \"watchonly\": <true>                                   , (boolean, optional, default: false) Stating whether matching outputs should be considered watched even when they're not spendable, only allowed if keys are empty\n"
             "      \"label\": <label>                                      , (string, optional, default: '') Label to assign to the address (aka account name, for now), only allowed with internal=false\n"
+            "      \"keypool\": <false>                                    , (boolean, optional, default: false) Stating whether imported public keys should be added to the keypool for when users request new addresses. Only allowed when wallet private keys are disabled\n"
```

Comment:
15426: we don't have this yet. 15426 requires at least 14075 (which requires others it seems), so it's too early - should drop 15426 for now

Reply:
dropped 15426